packages/core/src/Stores/client-store.js

```diff
@@ -669,6 +669,9 @@ export default class ClientStore extends BaseStore {
         this.registerReactions();
         this.setIsLoggingIn(false);
         this.setInitialized(true);
+
+        // upon logging in, check if account that is logged into is demo to make sure account tab is corect upon toggling
+        if (this.is_virtual) this.root_store.ui.setAccountSwitcherTabIndex(1);
```

Comment:
Can't this check be done in the `AccountSwitcher` component itself?

Reply:
It's so the `tab_index` for `AccountSwitcher` is set before toggling the *AccountSwitcher* `dialog`.

Reply:
As the tab state was moved from UI store, so was this.